scripts/make-multitenant-cluster.sh

```diff
@@ -26,6 +26,9 @@ yq eval "$query" "$source" > "$target"
 # Remove the aadpodidbinding label - this is only needed for communicating to ARM
 yq eval -i "del(select($deployment) | .spec.template.metadata.labels.aadpodidbinding)" "$target"
 
+# Edit the deployment to turn off leader election for the webhook only pod, as the webhooks don't wait for leader election anyway.
```

Comment:
I'm pretty sure that the parameter was only carried across for inertia/superstition reasons!